Sources/CardPayments/CardClient.swift

```diff
@@ -37,19 +37,28 @@ public class CardClient: NSObject {
         self.vaultAPI = vaultAPI
         self.webAuthenticationSession = webAuthenticationSession
     }
-    
-    @_documentation(visibility: private)
+
+    /// Updates a setup token with a payment method. Performs
+    /// 3DS verification if required. Check verification status with  /v3/vault/setup-token/{id} in your server.
```

Comment:
```suggestion
    /// 3DS verification if required. Check verification status with /v3/vault/setup-token/{id} in your server.
```

Reply:
> Check verification status with /v3/vault/setup-token/{id} in your server.

I am a bit concerned with this phrasing that merchants will think `vault()` causes the SDK to make a client-side request to `/v3/vault/setup-token/{id}`.

Reply:
Oh. I didn't think about that. Maybe "Check your verification status with .. after SDK returns with CardVaultResult"?

Reply:
Yeah we should make it as clear as we possibly can. Ultimately the developer.paypal docs will be the best place to explain the big-picture client / server interactions.